services/harmony/app/util/cmr.ts

```diff
@@ -665,9 +670,10 @@ async function _cmrPostBody(
  * @param token - Access token for user request
  * @returns The variable search results
  */
-export async function getAllVariables(
+export async function _getAllVariables(
```

Comment:
I think you want to remove all the `export` directives form all these functions you have made private.